static/js/components/search/SemesterRefinementOption.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from "react"
+import R from "ramda"
+
+export default class SemesterRefinementOption extends React.Component {
+  props: {
+    label: string,
+    active: boolean,
+    onClick: Function,
+    count: number
+  }
+
+  render() {
+    const { active, onClick, count, label } = this.props
+    const activeClass = () => (active ? "is-active" : "")
```

Comment:
I think you could just declare this as a boolean, unless I'm missing some reason why it needs to be called below as a function